### ISSUE TITLE
dssp can not be installed

### ISSUE THREAD

**Issue by igortru** [salilab/conda-recipes#14]
site http://swift.cmbi.ru.nl/gv/dssp/ is not accessible.

**Comment by benmwebb** [comment]
Yes, looks like the dssp people let their old web and FTP sites break, nothing we can do about that, sorry.

This package is a build of DSSP 3, not the latest DSSP 4 with mmCIF support. If DSSP 3 is what you need, we do maintain our own mirror at https://salilab.org/dssp/, so you could modify meta.yaml to point to that.

**Comment by igortru** [comment]
I have installed dssp from salilab successfullybut it crashed on my linux,only full recompile with boost installationresolved all problems with dssp in trRosettaX-Single.Sent from my iPhoneOn Dec 20, 2022, at 4:08 PM, Ben Webb ***@***.***> wrote:﻿
Yes, looks like the dssp people let their old web and FTP sites break, nothing we can do about that, sorry.
This package is a build of DSSP 3, not the latest DSSP 4 with mmCIF support. If DSSP 3 is what you need, we do maintain our own mirror at https://salilab.org/dssp/, so you could modify meta.yaml to point to that.
